tests/components/androidtv/test_media_player.py

```diff
@@ -1202,11 +1203,12 @@ async def test_exception(hass: HomeAssistant) -> None:
         assert state.state == STATE_OFF
 
         # When an unforeseen exception occurs, we close the ADB connection and raise the exception
-        with patchers.PATCH_ANDROIDTV_UPDATE_EXCEPTION, pytest.raises(Exception):
+        with patchers.PATCH_ANDROIDTV_UPDATE_EXCEPTION:
             await async_update_entity(hass, entity_id)
-            state = hass.states.get(entity_id)
-            assert state is not None
-            assert state.state == STATE_UNAVAILABLE
+
```

Comment:
Could you also test that the log contained the error that you expected?

Reply:
Added check